lib/css/modules/_styleTweaks.scss

```diff
@@ -144,3 +144,15 @@ blockquote.twitter-tweet {
 		display: none;
 	}
 }
+
+.res-styleTweaks-flairEmojiAsText-always,
+.res-styleTweaks-flairEmojiAsText-nosubstyle body.res-srstyle-disabled {
```

Comment:
You can squish this together, since they're both listed inside `<body class="...">`

```css
.res-styleTweaks-flairEmojiAsText-nosubstyle.res-srstyle-disabled
```